### PR TITLE
IAC adding config for ia-case-api to aat

### PR DESCRIPTION
**Pull request by raghera**
Adding IAC config for ia-case-api for AAT

### CODE REVIEW

File: k8s/perftest/common/ia/case-notifications-api.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      memoryRequests: '768Mi'
+      cpuRequests: '500m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1000m'
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "4315969c-54e6-43eb-b1c7-2be6fd411b82"
+      image: hmctspublic.azurecr.io/ia/case-notifications-api:prod-8c529b3f
+      ingressHost: ia-case-notifications-api-perftest.service.core-compute-perftest.internal
+      environment:
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
```
Comment (timja) [inline k8s/perftest/common/ia/case-notifications-api.yaml]: if you update your docker base image you can remove this

File: k8s/perftest/common/ia/case-notifications-api.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ia-case-notifications-api
+    version: 0.0.2
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '768Mi'
```
Comment (timja) [inline k8s/perftest/common/ia/case-notifications-api.yaml]: if you update your docker base image you can remove the this and the next 3 lines

File: k8s/perftest/common/ia/case-notifications-api.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      cpuRequests: '500m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1000m'
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "4315969c-54e6-43eb-b1c7-2be6fd411b82"
```
Comment (timja) [inline k8s/perftest/common/ia/case-notifications-api.yaml]: if you update chart-java and add bootstrap.yaml config you'll be able to pull this  from vault so you don't need to set it per  env

File: k8s/perftest/common/ia/case-notifications-api.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      image: hmctspublic.azurecr.io/ia/case-notifications-api:prod-8c529b3f
+      ingressHost: ia-case-notifications-api-perftest.service.core-compute-perftest.internal
```
Comment (timja) [inline k8s/perftest/common/ia/case-notifications-api.yaml]: this  can be set in your charts values.yaml as:
```
ia-case-notifications-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

saves having to update it in every env

File: CODEOWNERS [hunk 5]
```diff
@@ -46,7 +46,9 @@ k8s/aat/common/money-claims/claim-store.yaml @hmcts/cmc
 ## I&A
 k8s/**/common/ia/namespace.yaml @hmcts/immigration-asylum
 k8s/**/common/ia/identity.yaml @hmcts/immigration-asylum
-k8s/aat/common/ia/case-documents-api.yaml @hmcts/immigration-asylum
+k8s/**/common/ia/case-api.yaml @hmcts/immigration-asylum
```
Comment (timja) [inline CODEOWNERS]: ah these should be aat until you have prod approval, only case-documents-api is approved currently

Comment (raghera) [reply CODEOWNERS]: Ok, have updated those.

File: CODEOWNERS [hunk 6]
```diff
@@ -46,7 +46,9 @@ k8s/aat/common/money-claims/claim-store.yaml @hmcts/cmc
 ## I&A
 k8s/**/common/ia/namespace.yaml @hmcts/immigration-asylum
 k8s/**/common/ia/identity.yaml @hmcts/immigration-asylum
+k8s/aat/common/ia/case-api.yaml @hmcts/immigration-asylum
 k8s/aat/common/ia/case-documents-api.yaml @hmcts/immigration-asylum
```
Comment (timja) [inline CODEOWNERS]:
```suggestion
k8s/**/common/ia/case-documents-api.yaml @hmcts/immigration-asylum
```

**Pull request merged**